package.json

```diff
@@ -38,6 +43,7 @@
     "clean-docs": "rimraf website/docs && rimraf docs-api-json",
     "generate-d-ts": "npm run clean-docs && api-extractor run --local --verbose",
     "generate-docs": "npm run generate-d-ts && api-documenter markdown -i docs-api-json -o website/docs && node utils/remove-tag.js",
+    "generate-pkg-json": "cp package.esm.json lib/esm/package.json",
```

Comment:
nit: but perhaps `echo '{"type":"module"}' > lib/esm/package.json` would work without the need for an extra file?